docs/WEBPACK.md

```diff
@@ -83,3 +83,50 @@ if (ENV === "development") {
 	/* the code inside this block will be executed only in development */
 }
 ```
+
+
+#### [HtmlElementWebpackPlugin](https://www.npmjs.com/package/html-elements-webpack-plugin "HtmlElementWebpackPlugin")
+
+This plugin allows the webpack plugin to append head elements during the creation of index.html. 
+
+To use it, you'll have to create the `head-config.common.js` file to specify the links and meta you want to import in your index.html. 
```

Comment:
Where in the project do we have to create the head-config.common.js file ?

Reply:
And done :)